src/aabb/mod.rs

```diff
@@ -23,28 +23,24 @@
 pub use self::aabb2::Aabb2;
 pub use self::aabb3::Aabb3;
 
-use std::cmp::{PartialOrd, Ordering};
+use std::cmp::{Ordering, PartialOrd};
 
-use cgmath::{Point2, Point3, BaseNum};
+use cgmath::{BaseNum, Point2, Point3};
 use cgmath::prelude::*;
 
 mod aabb2;
 mod aabb3;
 
 pub(crate) fn min<S: PartialOrd + Copy>(lhs: S, rhs: S) -> S {
     match lhs.partial_cmp(&rhs) {
-        Some(Ordering::Less) |
-        Some(Ordering::Equal) |
-        None => lhs,
+        Some(Ordering::Less) | Some(Ordering::Equal) | None => lhs,
```

Comment:
this doesn't look better to me

Reply:
There's no config switch for this in rustfmt unfortunately. And I personally think the new version is better, easier to find the right hand of the match arm imho.